frontend/dockerfile/dockerfile2llb/convert.go

```diff
@@ -30,6 +30,7 @@ import (
 	"github.com/moby/buildkit/util/apicaps"
 	binfotypes "github.com/moby/buildkit/util/buildinfo/types"
 	"github.com/moby/buildkit/util/gitutil"
+	"github.com/moby/buildkit/util/schema1"
```

Comment:
This isn't technically `schema1`. Schema1 is that `V1Compatibility` mess. Put it under `exporter/containerimage/image`

Reply:
According to your suggestion, move `Image` definition to package `exporter/containerimage/image`, PTALA.